server/methods/core/cart.js

```diff
@@ -511,8 +511,22 @@ Meteor.methods({
     if (cart.userId !== this.userId) {
       throw new Meteor.Error(403, "Access Denied");
     }
-    const order = Object.assign({}, cart);
+
+    let orders = [];
+    let order = Object.assign({}, cart);
+    delete order.items;
+    delete order.shopId;
     const sessionId = cart.sessionId;
+    const itemsByShop = _.groupBy(cart.items, "shopId");
+    for (var item in itemsByShop) {
```

Comment:
I think that splitting items by shop should also probably be a separate PR, though it does relate directly to Marketplace, it's not related to Stripe Connect